client/src/components/App.js

```diff
@@ -1,15 +1,16 @@
 import React from "react";
-import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import { Router, Route, Switch } from "react-router-dom";
 
 import Register from "../pages/Register";
 import Login from "../pages/Login";
 import Messenger from "../pages/Messenger";
+import history from "../history";
```

Comment:
hm any reason why you decided to use a history library rather than utilizing the history hook through react-router-dom? 

https://reactrouter.com/web/api/Hooks

Reply:
I think the tutorial I watched was using class components. I didn't know there was a history hook through react-router-dom, going to implement this now!